judge/.gitignore

```diff
@@ -0,0 +1,3 @@
+.env
```

Comment:
Update this file to ignore db.sqlite3 and *.pyc files. These files are generated every time the app runs.

Reply:
Update this and I will merge the changes.

Reply:
done @ExpressHermes